features/step_definitions/report_steps.rb

```diff
@@ -28,3 +28,22 @@
 
   add_named_record name, create(:test_payload, options)
 end
+
+Given /^contextualized test payload (.+) sent by (.+) for version (.+) of project (.+) was used to generate report (.+?) with context:/ do |name,runner_name,version_name,project_name,report_name,context|
```

Comment:
`contextualized test payload ... with context`

Is "contextualized" really necessary here? It's kind of a pleonasm.